test/capybara_screenshot_diff/dsl_test.rb

```
@@ -0,0 +1,135 @@
+        assert_match(/our_screenshot'/, CapybaraScreenshotDiff.assertions[0].caller.first)
+        assert_equal snap.full_name, CapybaraScreenshotDiff.assertions[0].name
+
+        our_screenshot(snap.full_name, 1)
+        assert_equal 2, CapybaraScreenshotDiff.assertions.size
+        assert_match(
+          %r{/dsl_test.rb:.*?test_skip_stack_frames},
+          CapybaraScreenshotDiff.assertions[1].caller.first
+        )
+        assert_equal snap.full_name, CapybaraScreenshotDiff.assertions[1].name
+      end
+    end
```

RuboCop (1.75.5)

[convention] Assignment Branch Condition size for test_skip_stack_frames is too high. [<1, 33, 0> 33.02/23]

(Metrics/AbcSize)